src/main/java/org/jabref/gui/preferences/NameFormatterTabView.java

```diff
@@ -0,0 +1,104 @@
+package org.jabref.gui.preferences;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.TableColumn;
+import javafx.scene.control.TableView;
+import javafx.scene.control.TextField;
+import javafx.scene.control.cell.PropertyValueFactory;
+import javafx.scene.control.cell.TextFieldTableCell;
+import javafx.scene.input.KeyCode;
+
+import org.jabref.Globals;
+import org.jabref.gui.actions.ActionFactory;
+import org.jabref.gui.actions.SimpleCommand;
+import org.jabref.gui.actions.StandardActions;
+import org.jabref.gui.help.HelpAction;
+import org.jabref.gui.icon.IconTheme;
+import org.jabref.gui.util.ValueTableCellFactory;
+import org.jabref.logic.help.HelpFile;
+import org.jabref.logic.l10n.Localization;
+import org.jabref.preferences.JabRefPreferences;
+
+import com.airhacks.afterburner.views.ViewLoader;
+
+public class NameFormatterTabView extends AbstractPreferenceTabView implements PreferencesTab {
+
+    @FXML private TableView<NameFormatterItemModel> formatterList;
+    @FXML private TableColumn<NameFormatterItemModel, String> formatterNameColumn;
+    @FXML private TableColumn<NameFormatterItemModel, String> formatterStringColumn;
+    @FXML private TableColumn<NameFormatterItemModel, String> actionsColumn;
+    @FXML private TextField addFormatterName;
+    @FXML private TextField addFormatterString;
+    @FXML private Button formatterHelp;
+    @FXML private Button addFormatter;
+
+    public NameFormatterTabView(JabRefPreferences preferences) {
+        this.preferences = preferences;
+
+        ViewLoader.view(this)
+                .root(this)
+                .load();
+    }
+
+    public void initialize () {
+        NameFormatterTabViewModel nameFormatterTabViewModel = new NameFormatterTabViewModel(dialogService, preferences);
+        this.viewModel = nameFormatterTabViewModel;
+
+        formatterList.setEditable(true);
+
+        formatterNameColumn.setSortable(true);
+        formatterNameColumn.setReorderable(false);
+        formatterNameColumn.setCellValueFactory(new PropertyValueFactory<>("name"));
```

Comment:
Is there a reason not to use `ValueTableCellFactory` here (and below?)

Reply:
Do you mean instead of PropertyValueFactory<>() or instead of TextFieldTableCell.forTableColumn()?

Reply:
It's an editable cell I think?

Reply:
I think it should look like
https://github.com/JabRef/jabref/blob/8520833e684aa9e305dd8745e45696ffd6d984fb/src/main/java/org/jabref/gui/openoffice/ManageCitationsDialogView.java#L63-L64
or like
https://github.com/JabRef/jabref/blob/8520833e684aa9e305dd8745e45696ffd6d984fb/src/main/java/org/jabref/gui/openoffice/ManageCitationsDialogView.java#L66-L67

Reply:
Well, I removed the PropertyValueFactory, but I think the ValueTableCellFactory does not suppert editable Cells.